examples/esp32/partitions.csv

```diff
@@ -1,5 +1,5 @@
 # Name,   Type, SubType, Offset,  Size, Flags
 # Note: if you have increased the bootloader size, make sure to update the offsets to avoid overlap
-nvs,      data, nvs, 0x9000    ,        0x8000,
-phy_init, data, phy, 0x11000    ,        0x1000,
-factory,  app,  factory, 0x30000,       0x3D0000,
+nvs,      data, nvs, 0x9000    ,        0x6000,
+phy_init, data, phy, 0xf000    ,        0x1000,
```

Comment:
[q] how often should we check that the updated compiled binary fits in the factory partition?

Reply:
We should be mindful we don't want the binary to bloat even in Debug mode